src/Storages/ObjectStorage/DataLakes/DeltaLake/KernelHelper.cpp

```diff
@@ -87,20 +94,25 @@ namespace S3AuthSetting
     extern const S3AuthSettingsString region;
 }
 
-DeltaLake::KernelHelperPtr getKernelHelper(const StorageObjectStorage::ConfigurationPtr & configuration)
+DeltaLake::KernelHelperPtr getKernelHelper(
+    const StorageObjectStorage::ConfigurationPtr & configuration,
+    const ObjectStoragePtr & object_storage)
 {
     switch (configuration->getType())
     {
         case DB::ObjectStorageType::S3:
         {
             const auto * s3_conf = dynamic_cast<const DB::StorageS3Configuration *>(configuration.get());
             const auto auth_settings = s3_conf->getAuthSettings();
```

Comment:
Seems like `auth_settings` no longer needed, right?

Reply:
Fixed in [4aedb21](https://github.com/ClickHouse/ClickHouse/pull/77661/commits/4aedb219ec4ae5f0068cdb45e4470e55eb693a80)